packages/fx-core/src/plugins/resource/function/index.ts

```diff
@@ -143,6 +143,14 @@ export class FunctionPlugin implements Plugin {
     return res;
   }
 
+  public async updateArmTemplates(ctx: PluginContext): Promise<FxResult> {
```

Comment:
Seems you include the same change in two PRs?

Reply:
sorry, will close this PR